pkg/kubelet/stats/helper.go

```diff
@@ -127,19 +126,26 @@ func cadvisorInfoToNetworkStats(name string, info *cadvisorapiv2.ContainerInfo)
 	if !found {
 		return nil
 	}
+
+	var rxBytes uint64
+	var rxErrors uint64
+	var txBytes uint64
+	var txErrors uint64
+
 	for _, inter := range cstat.Network.Interfaces {
-		if inter.Name == network.DefaultInterfaceName {
-			return &statsapi.NetworkStats{
-				Time:     metav1.NewTime(cstat.Timestamp),
-				RxBytes:  &inter.RxBytes,
-				RxErrors: &inter.RxErrors,
-				TxBytes:  &inter.TxBytes,
-				TxErrors: &inter.TxErrors,
-			}
-		}
+		rxBytes += inter.RxBytes
+		rxErrors += inter.RxErrors
+		txBytes += inter.TxBytes
+		txErrors += inter.TxErrors
```

Comment:
I don't think that summing these statistics over all interfaces is the right approach. For instance, you probably don't care about how much traffic is going over loopback. I think the right solution is to either make it possible to select the monitored interface (see https://github.com/kubernetes/kubernetes/issues/28407), or modify the API to include a list of network stats.

Reply:
> For instance, you probably don't care about how much traffic is going over loopback.

Actually, when requesting cadvisor to return network interface values, [network interface with name contains `lo`, `veth` and `docker` will be ignored.](https://github.com/google/cadvisor/blob/master/container/libcontainer/helpers.go#L162-L227). So, this should not be the problem.

Reply:
@tallclair PTAL.

Reply:
ping @tallclair

Reply:
OK, I could be sold on this, but I would prefer to also offer the per-interface stats through another method. Perhaps someone from @kubernetes/sig-network-pr-reviews can chime in about how common multiple-interfaces are, and any other potential side effects to combining the stats.

Reply:
Another example, I just looked at a random node on a GKE cluster and it has a cbr0 device (container bridge), which should definitely be counted separately from the eth0 stats.

Reply:
>  I just looked at a random node on a GKE cluster and it has a cbr0 device (container bridge), which should definitely be counted separately from the eth0 stats.

@tallclair This should be done in cAdvisor.